wsdl-cli/src/main/java/io/ballerina/wsdl/cli/ErrorMessages.java

```diff
@@ -0,0 +1,24 @@
+/*
+ *  Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com)
+ *
+ *  WSO2 LLC. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package io.ballerina.wsdl.cli;
+
+public class ErrorMessages {
+    public static final String MISSING_WSDL_PATH = "A WSDL definition path is required to generate the " +
```

Comment:
This error message can be moved to the place which currently uses this. In future if we have more than one error message we can group them into a common place, but ATM we don't need it.